README.md

```diff
@@ -0,0 +1,91 @@
+- Use ImageKit's [upload API](https://docs.imagekit.io/api-reference/upload-file-api/server-side-file-upload) to set `tags`, `useUniqueFileName`, and `folder`.
+
+## Installation
+
+### Using yarn
+
+- Go into your Ghost project's root directory.
+- Install the adapter:
+
+```bash
+yarn add ghost-imagekit-store
+cp node_modules/ghost-imagekit-store/dist/src/index.js content/adapters/storage/imagekit/
+```
+
+- Configure Ghost's config file.
+
+### Install on Docker
+
+Here's an example of using this adapter with a containerized Ghost:
+
+```Dockerfile
+FROM ghost:5-alpine as imagekit
+RUN apk add g++ make python3
+RUN su-exec node yarn add ghost-imagekit-store
+
+FROM ghost:5-alpine
+COPY --chown=node:node --from=imagekit $GHOST_INSTALL/node_modules $GHOST_INSTALL/node_modules
+COPY --chown=node:node --from=imagekit $GHOST_INSTALL/node_modules/ghost-imagekit-store/dist/src/index.js $GHOST_INSTALL/content/adapters/storage/imagekit/index.js
+# Here, we use the Ghost CLI to set some pre-defined values.
+RUN set -ex; \
+    su-exec node ghost config storage.active imagekit; \
+    su-exec node ghost config storage.imagekit.uploadOptions.useUniqueFileName true; \
+    su-exec node ghost config storage.imagekit.uploadOptions.folder /ghost/blog;
+```
+
+Make sure the content path is correctly set in the Ghost configuration:
+
+```json
+"paths": {
+    "contentPath": "/var/lib/ghost/content/"
+}
+```
+
+Ensure that the `GHOST_CONTENT` environment variable is set to the same value as that of `paths.contentPath` in your Ghost app.
+
+## Configuration
+
+Check out [configuration.json.dist](./configuration.json.dist) for a complete example.
+
+- Make sure [Ghost Image Optimization](https://ghost.org/docs/config/#image-optimisation) is disabled.
+- The optional `enableDatedFolders` setting allows uploading images into dated sub-directories (like the default Ghost Local Storage Adapter). It is `true` by default.
+- The `auth` property is used to configure your ImageKit account's credentials and URL endpoint. You can find them in the [dashboard](https://imagekit.io/dashboard/developer/api-keys).
+- The `uploadOptions` property allows you to configure ImageKit's upload options. You can configure `useUniqueFileName`, `folder`, and `tags`.
+
+### Recommended configuration
+
+- `uploadOptions.useUniqueFileName = true` Ghost's local storage adaptor handles duplicate file names automatically. When uploading a file to a location where a file with the same name already exists, ImageKit creates a new file version. In order to avoid this, `useUniqueFileName` can be set to `true`.
+- `uploadOptions.tags = ["travel", "discover"]` if you want to add associations of tags to your uploaded images.
+- `uploadOptions.folder = "/ghost/blog"` allows to upload all your images into a specific directory in your ImageKit media library. By default, files are uploaded to the root of media library.
+
```

Comment:
@abhinavr4 Can we add the sample for config file here in the README as well?  It would help the user visualize the config options immediately instead of them having to check the `configuration.json.dist` file first. Simply like this:

```
{
    "storage": {
        "active": "imagekit",
        "imagekit": {
            "enableDatedFolders": true,
            "auth": {
                "privateKey": "your-private-key",
                "publicKey": "your-public-key",
                "urlEndpoint": "https://ik.imagekit.io/your-imagekit-id"
            },
            "uploadoptions": {
                "useUniqueFileName": false,
                "folder": "/sample",
                "tags": ["blog"]
            }
        }
    },
    "imageOptimization": {
        "resize": false
    }
}
```

Reply:
done